.phan/config.php

```diff
@@ -32,7 +32,6 @@
 		"PhanNonClassMethodCall",
 		"PhanTypeArraySuspicious",
         "PhanTypeSuspiciousStringExpression",
-        "PhanContinueTargetingSwitch",
 	],
```

Comment:
Everything looks correct from what I gather. 
My only question is if "PhanContinueTargetingSwitch" should be removed. It seems like a good way to warn future mistakes of using continue instead of break.

Reply:
It's hard to tell because GitHub doesn't show the full context, but this is a list of ignored types that it's being removed from (ie. to make it warn of mistakes in the future), not a list of enforced types.

Reply:
@driusan Okay then everything looks good to me.